examples/workflows/custom_base_node/nodes/mock_networking_client.py

```diff
@@ -33,6 +33,9 @@ class Outputs(BaseNode.Outputs):
         response: dict
 
     def run(self) -> BaseNode.Outputs:
+        if not self.state.chat_history:
+            self.state.chat_history = []
```

Comment:
this raises the idea of how we should explore default field factories, but punting that for now